.gitignore

```diff
@@ -90,6 +90,12 @@ ENV/
 env.bak/
 venv.bak/
 
+# direnv file
+.envrc
+
+# IDEA ide
+.idea/
+
```

Comment:
I copied this change to https://github.com/adamtheturtle/sphinx-substitution-extensions/pull/104 and merged it so that this repo can be friendlier without polluting this PR.